src/collective/cover/tests/test_textlinessortable_widget.py

```diff
@@ -158,20 +158,65 @@ def test_extract(self):
         self.assertDictEqual(extracted_value, expected)
 
     def test_utf8_custom_data(self):
-        obj = self.portal['my-image']
-        obj.setDescription('áéíóú')
+        obj1 = self.portal['my-image']
+        obj1.setDescription('áéíóú')
+        obj2 = self.portal['my-image3']
```

Comment:
IIUC my-image2 will do, no?

Reply:
IIUC, whether you use the title of the image object or set a custom title you will be hitting different code paths, which is why the existing tests did not catch this issue.  That is why I created a `my-image3` which has both a title and description containing accented letters.
I don't understand the robot test failures, though.
Maybe I'll try it without `my-image3` first to see if the failures go away.